### ISSUE TITLE
Control scroll in file explorer

### ISSUE THREAD

**Issue by alexrosenfeld10** [microsoft/vscode#68713]
This is a minor feature request, I happened to notice it missing.

In the editor window, control + arrow keys scrolls down instead of moves your cursor down. The same action in the file explorer doesn't do anything.

As someone who uses keyboard shortcuts for the majority of their workflow, it would be a nice to have.

Thanks!

**Comment by isidorn** [comment]
This should work for all our trees in case we want to add it. Forwarding to @joaomoreno

**Comment by devilirium** [comment]
I've decided to check the current behavior for Ctrl + Down and Ctrl + Up in VS2017, Notepad++ and the old school Notepad. In VS and Notepad++ I've noticed the same scrolling that VS Code has instead of cursor movement. In Notepad the shortcut seems to be undefined, it does't do anything. 

Based on these, I think it might be confusing to change the current behavior for users that are coming from a different editor.
